Project.toml

```diff
@@ -1,9 +1,10 @@
 name = "ChainRules"
 uuid = "082447d4-558c-5d27-93f4-14fc19e9eca2"
-version = "1.61.0"
+version = "1.61.1"
 
 [deps]
 Adapt = "79e6a3ab-5dfb-504d-930d-738a2a938a0e"
+AxisArrays = "39de3d68-74b9-583c-8d2d-e117c070f3a9"
```

Comment:
IMO ChainRules should not depend on AxisArrays.

Reply:
I don't have any insight on the merits for or against this. But what is your suggestion?

Reply:
AFAICT it has been a general policy to not accept such dependencies, see e.g. https://github.com/JuliaArrays/FillArrays.jl/pull/153#issuecomment-900188437

Reply:
An extension to FillArrays is also out of the question?

Reply:
The PR was before extensions existed, so I have been thinking for a while one should try again with an extension. I managed to get in an extension on PDMats recently, so I think it seems likely that it would be approved.

Reply:
Making that FillArrays PR into an extension there would be great.